sklearn/utils/tests/test_random.py

```diff
@@ -180,3 +183,24 @@ def test_random_choice_csc_errors():
     class_probabilities = [np.array([0.5, 0.6]), np.array([0.6, 0.1, 0.3])]
     assert_raises(ValueError, random_choice_csc, 4, classes,
                   class_probabilities, 1)
+
+
+@pytest.mark.parametrize("low,high,base", [
+    (-1, 0, 10),
+    (0, 2, np.exp(1)),
+    (-1, 1, 2),
+])
+def test_log_uniform(low, high, base):
+    rv = loguniform(low, high, base=base)
+    rvs = rv.rvs(size=100)
```

Comment:
You haven't asserted size. We should perhaps also check support for sizes in higher dimension, even if that's not applicable to grid search